MetaMask/metamask-extension
# Don't show "Unavailable on this network" if Swaps button is enabled

## Description
Don't show "Unavailable on this network" if Swaps button is enabled. 

Even with the `disabled` prop on the `Tooltip` component, UI was showing the "Unavailable on this network" text (as a div title) when a user hovered over the Swaps button. This PR makes sure that we don't pass any Tooltip title if a tooltip is disabled (e.g. when a network is enabled for Swaps).

## Screenshots
Before with enabled Swaps button:
![image](https://user-images.githubusercontent.com/80175477/139285385-effb3dfa-de6b-4215-9498-6007dcfe98b9.png)

Before with disabled Swaps button:
![image](https://user-images.githubusercontent.com/80175477/139285562-b418c870-fa25-4021-93b0-39c0bab88ad2.png)

After with enabled Swaps button:
![image](https://user-images.githubusercontent.com/80175477/139285497-5699f392-8633-49db-87eb-d46aa139c3aa.png)

After with disabled Swaps button:
![image](https://user-images.githubusercontent.com/80175477/139285629-889e2901-953c-41d6-b491-855c146f13b9.png)

## Manual testing steps:
  - Hover over the `Swaps` button on Ethereum. No tooltip is displayed
  - Hover over the `Swaps` button on Kovan. Tooltip is displayed
